pkg/releasetesting/environment.go

```diff
@@ -31,38 +32,47 @@ type Environment struct {
 	Timeout    int64
 }
 
-func streamRunning(name string, stream services.ReleaseService_RunReleaseTestServer) error {
+// EnvironmentStreamer implements Streamer functionality
+type EnvironmentStreamer struct {
```

Comment:
I suggest using the Environment struct instead of creating an EnvironmentStreamer and hanging the stream functions off of that. In some stream functions, you'll need more than just Environment.Stream anyway. For example, in the StreamFailed function, you also need namespace. This way, you won't have to pass it in.